components/statuslight/stories/statuslight.stories.js

```diff
@@ -1,7 +1,13 @@
 import { html } from "lit";
+import { styleMap } from "lit/directives/style-map.js";
+import { when } from "lit/directives/when.js";
+import { Template as Typography } from "@spectrum-css/typography/stories/template.js";
 
 import { Template } from "./template";
 
+/**
+ * Status lights describe the condition of an entity. They can be used to convey semantic meaning, such as statuses and categories.
+ */
```

Comment:
Do any of these controls need descriptions? It's only "size," "variant," and "label" so my gut says no. Those all seem pretty self-explanatory.

Reply:
I think the component description is okay right now, but I do think this will also be needed in the docs migration, that will likely go into main first.

Yeah, I don't think those controls need a description. "Variant" _might_ benefit from one, to say that the variant changes the color and the list of variants includes both semantic and non-semantic variants?

Reply:
Here's the description for "variant:" https://github.com/adobe/spectrum-css/pull/2818/commits/ee5dacb11514bb47dc0eb96f8ffa2a5f04fe33b2